translation from c00/30/16 to 20 (#66)
IllusionMods/HS2-Translation: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/19/1/translation.txt

//
// Dumped for Honey Select 2 v1.1.3 by Text Dump v1.4.1
//
急に腰が抜けてしまいまして……だらしなくて、すみません。=I suddenly felt my back give. I'm sorry for being so clumsy.
//っ……こんな時にもよおすとは……恥の上塗りです……=
……日頃から腰を振っていたせいでしょうか。まったく力が入りません。=...I wonder if it's because I've been pumping my hips daily. I simply can't move them at all.
//ここへきて尿意が……私はとうとう天から見放されたようです……=
今、あなたが救いの神に見えます。腰を痛めてしまったので、力を貸していただけませんか？=Right now, you look like a knight in shining armor. I hurt my back, so could you please help me?
//……すみません。理由を明かすことはできませんが、一刻も早く助けていただけないでしょうか。お願い致します。=

> 突然、腰に力が入らなくなり……もし襲うつもりでしたら、そっとしていただけないでしょうか。=I suddenly lost strength in my lower back. If you intend to use me, could you please do it gently?

Reviewer: ```suggestion
突然、腰に力が入らなくなり……もし襲うつもりでしたら、そっとしていただけないでしょうか。=I threw my back out... If you intend to use me, please be gentle.
```

Author: yeah, as expected, english has a lot of awkward phrases for back pains :P
